### DISCUSSION TITLE
Is typescript required?

### DISCUSSION THREAD

**Discussion by GerritKuilder** [nuxt/nuxt#18016] [Questions]
I just ran the new nuxt3 and it looking good, I am waiting for a stable/beta release of vuetify 3.

I might have a hard time getting the basics up as nuxt 2 used to do that for me: ie ask for a css framework to be installed and you were up and running.
(oh and if somebody has some basic nuxt3/vuetify3 set up that can be tested/ get my feet wet with I am willing to try)

I am mainly wondering (as I only looked at  and ino it for a few hours) if I am able to continue to use plain javascript in the vue script parts?

Please note, I have no opinion on typescript, I just don't have time to learn/look into typescript right now.

Kind Regards,

Gerrit Kuilder

**Comment by danielroe** [comment]
Yes, you can continue to use plain JavaScript - no need to use the `.ts` extension. (And indeed, even if you were to use a `.ts` extension, you are free to write plain JS in a `.ts` file.)
